blog/_src/posts/2017-06-05-trip-report-pliss-2017.md

```diff
@@ -0,0 +1,130 @@
+    Title: Report: PLISS 2017
+    Date: 2017-06-05T15:47:59
+    Tags: pliss, event, by Ming-Ho Yee
+
+Two weeks ago, I attended the first [Programming Language Implementation Summer
+School][PLISS], held in beautiful Bertinoro, Italy.
+
+The goal of PLISS was "to prepare early graduate students and advanced
+undergraduates for research in the field," and I think it successfully
+accomplished that. There were many talks in a variety of areas, such as
+just-in-time compilers, garbage collection, static analysis, and distributed
+systems. But PLISS was more than just a series of talks: PLISS provided an
+environment for interacting with other students, as well as senior researchers.
+
+<!-- more -->
+
+## The Talks
+
+With the amount of technical content at PLISS, there was easily something for
+everyone. Some of the lectures included hands-on exercises where we worked on
+JITs. One lecture dived into the operational semantics of a distributed system,
+so we could reason about different weak consistency models. Another lecture gave
+us a whirlwind tour of abstract interpretation.
```

Comment:
Care to link to the lectures (if that makes sense) or give the lecturers names (to serve as pointers) otherwise?